source/contributing/implementation/debugging/faq.rst

```diff
@@ -0,0 +1,37 @@
+==========================
+Frequently Asked Questions
```

Comment:
[quibble] Frequently Asked Questions are a great idea. The key word here though is "Questions". If you phrase them as such it fits better, or as dot points under a generic one ("Why won't IntelliJ behave?").